### DISCUSSION TITLE
Can I get opa parameters inside rego ?

### DISCUSSION THREAD

**Discussion by vas-rus** [open-policy-agent/community#256] [OPA and Rego]
I want to run different commands based on different env , how I can pass env value to rego file (running in test or prod) , so that decision can be made at runtime ?

**Comment by srenatus** [comment]
`opa.runtime().env` should have the env vars your server or command was started with. Try `opa eval -fpretty 'opa.runtime()'`.

But please be aware that this can make your policies harder to test: there's suddenly an environmental dependency that might not be obvious on the surface. _But_ it's possible to overwrite `opa.runtime()` in tests, though, too.
